h2o-algos/src/main/java/hex/aggregator/Aggregator.java

```diff
@@ -231,6 +230,7 @@ public void computeImpl() {
         }
         _job.update(1, "Done.");
         model.update(_job);
+        checkEffectiveParmsDoesNotContainAuto(model._effective_parms);
```

Comment:
this is also error-prone and could be simplified. I think we shoud ideally move this call to Model so that we do not need duplicate it in every model sub-class
can you think of a good place where to have this? is it after train or after init? 
ideally I would do the same for initEffectiveParams() maybe make that an overridable method in Model and call it at some defined place, also how about renaming that to computeEffectiveParams, to make it more clear?

Reply:
initEffectiveParams() will be easy. checkEffectiveParmsDoesNotContainAuto() should be in the end/ after hex.ModelBuilder.Driver#computeImpl(). can try to find some place for it